### ISSUE TITLE
fsModules.fsPersonGender needs to escape  quotes within name

### ISSUE THREAD

**Issue by thorntonjn** [fs-webdev/fs-modules#24]
Steven looked at it with me and discovered that  element.innerHtml changed the string so that the result in not escaped or html encoded. 

To reproduce go to login to familysearch.org/tree
and go to 
familysearch.org/tree/?debug=true#tab=id&view=search&id=LJGD-2Q4&search=1&cb=1439326763216

**Comment by redbugz** [comment]
Everything appears to be correct to me on that URL. Can you be more specific as to the error. Also, I would be very surprised if innerHtml actually modified the escaping or encoding of your input string, is it actually modifying it, or just not honoring what was input? And how does the gender value impact the name value at all?

**Comment by thorntonjn** [comment]
If you login to familysearch and then go to the above URL you will see a search result with Mary Martha "Mattie" Fimple and an <a> tag with a data-cmd-data attribute with a string that should represent a parseable JSON object.  However the string is not vaild JSON because "Mattie" is not properly escaped.   

Steven and I traced into the fsPersonGender code that sets the value and we discovered that it changed from a properly formatted JSON string to something that was no longer valid after the innerHTML method was invoked.  Strange, I admit, but I think the string needs to somehow be escaped differently going into that call.  At least it needs to somehow be corrected.
